python/src/Sample.i

```diff
@@ -786,6 +786,26 @@ Sample(PyObject * pyObj)
   return new OT::Sample( OT::convert< OT::_PySequence_, OT::Sample>(pyObj) );  
 }
 
+static Sample BuildFromSequence(PyObject * pyObj, const OT::UnsignedInteger size, const OT::UnsignedInteger dimension)
```

Comment:
You can probably use Point type directly instead of PyObject.
I wonder if it should be moved to the c++ lib, as it could be actually needed there.

Reply:
Maybe indeed. I will move it to `c++`

Reply:
@jschueller If I understand your remarks, the idea is to implement `Sample::BuildFromPoint(Point &)`?

Reply:
yup!